keps/sig-node/2413-seccomp-by-default/README.md

```diff
@@ -378,26 +375,22 @@ _This section must be completed when targeting beta graduation to a release._
 
 - **How does this feature react if the API server and/or etcd is unavailable?**
 
+  It will make the feature not work as well.
+
 - **What are other known failure modes?**
-  For each of them, fill in the following information by copying the below template:
-
-  - [Failure mode brief description]
-    - Detection: How can it be detected via metrics? Stated another way:
-      how can an operator troubleshoot without logging into a master or worker node?
-    - Mitigations: What can be done to stop the bleeding, especially for already
-      running user workloads?
-    - Diagnostics: What are the useful log messages and their required logging
-      levels that could help debug the issue?
-      Not required until feature graduated to beta.
-    - Testing: Are there any tests for failure mode? If not, describe why.
+
+  None
```

Comment:
If the feature is disabled on kubelets, but the seccomp.type is set, what happens to the pod?  Are the values not honored?  That seems like a significant problem.

Reply:
If `seccomp.type` is `RuntimeDefault` and the feature is disabled then it will apply `RuntimeDefault`. I don't see any other failure modes than workload termination or misbehavior (described in the SLIs now). Do you have anything in mind?

Reply:
I wasn't sure if there were edges around specifying a seccomp type that doesn't exist.

Reply:
Cool, looks like we're then all set. :blush: